### PR TITLE
Release script - Build Release Assets

### PR DESCRIPTION
**Pull request by subpathdev**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
adding a release script, with this script I build the releases for the following architectures:
- armV6
- armV7
- x86_64

**Which issue(s) this PR fixes**:
xref: #944 

**Special notes for your reviewer**:
In a separate pr we can use the github release api (https://developer.github.com/v3/repos/releases/)

### CODE REVIEW

File: docs/guides/release.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# release binaries
+
+## Table of Contents
+* [Introduction](#intorduction)
+* [Generate](#generate)
+* [Add architecture](#add architecture)
+* [Remove architecture](#remove architecture)
+
+## Introduction
+In this file I will describe, how we can build the release binaries.
```
Comment (rohitsardesai83) [inline docs/guides/release.md]: In this file I will describe, how we can build the release binaries  -->

This section describes how can we build the release binaries.

File: docs/guides/release.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# release binaries
+
+## Table of Contents
+* [Introduction](#intorduction)
```
Comment (rohitsardesai83) [inline docs/guides/release.md]: intorduction -> introduction

File: docs/guides/release.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# release binaries
```
Comment (rohitsardesai83) [inline docs/guides/release.md]: release -> Release

File: docs/guides/release.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+- edge-$version-linux-x86\_64
+- edge-$version-linux-armV6
+- edge-$version-linux-armV7
+- kubeedge-$version-linux-x86\_64
+- kubeedge-$version-linux-armV6
+- kubeedge-$version-linux-armV7
+- edgesite-$version-linux-x86\_64
+- edgesite-$version-linux-armV6
+- edgesite-$version-linux-armV7
+- the checksum files to every builded binary package
```
Comment (rohitsardesai83) [inline docs/guides/release.md]: Can we create a single checksum file which is having the checksums for all the binaries separated by newlines ?
Maybe too many checksum files is not very user friendly ?

File: docs/guides/release.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+* [Remove architecture](#remove architecture)
+
+## Introduction
+In this file I will describe, how we can build the release binaries. 
+In the current state we will build the release for the following architectures:
+- armV6
+- armV7
+- x86\_64
+
+We are using a seperate release script on the build/tool directory.
```
Comment (rohitsardesai83) [inline docs/guides/release.md]: on -> in

File: build/tools/release.sh [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+	cp -r cloud/conf $kubeedge/cloud/
+
+	# copy keadm to the expected directories
+	cp keadm/keadm $keadm/
+
+	# copy to edgesite
+	cp edgesite/edgesite $edgesite/
+	cp -r edgesite/conf $edgesite/
+
+	tar cfv $(echo $keadm.tar) $keadm
```
Comment (rohitsardesai83) [inline build/tools/release.sh]: can we compress further and make all of them .tar.gz ?

File: docs/guides/release.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+## Generate
+To generate a current release you can build all the files from hand or using this script. You can execute from the root direcotry of this repository via this command:
```
Comment (rohitsardesai83) [inline docs/guides/release.md]: direcotry -> directory

File: docs/guides/release.md (hunk 4 above)
Comment (subpathdev) [reply docs/guides/release.md]: Yes of course, I will use checksum_$arch.txt as filename

File: build/tools/release.sh (hunk 6 above)
Comment (subpathdev) [reply build/tools/release.sh]: Yes of course

**Comment by sids-b** [comment]
/approve

/hold
ping @fisherxu for final review

**Pull request closed without merging**